namespace-provisioner/use-case7.hbs.md

```diff
@@ -0,0 +1,471 @@
+# Work with Git repositories in air-gapped with Namespace Provisioner
+
+This section provides instructions on configuring the Namespace Provisioner to utilize air-gapped
+Git repositories. This allows you to store GitOps-based installation files and platform
+operator templated resources intended for creation in your developer namespace within
+Tanzu Application Platform (TAP).
+
+
+## <a id= 'git-auth'></a>Git Authentication
+
+Authentication is established through a secret in the `tap-namespace-provisioning` namespace or
+an existing secret in another namespace referenced in the `secretRef` within `additional_sources`.
+For more details, refer to [Customize Installation of Namespace Provisioner](customize-installation.hbs.md).
+
+### Create the Git Authentication secret in tap-namespace-provisioning namespace
+
+The secrets for Git authentication allow the following keys: ssh-privatekey, ssh-knownhosts, username, and password. If ssh-knownhosts is not specified, Git does not perform strict host checking.
+
+The Git authentication secrets support the following keys: `ssh-privatekey`, `ssh-knownhosts`, `username`, and `password`. If `ssh-knownhosts` is not specified, Git does not perform strict host checking.
+
+>**Important:** In air-gapped environments or other scenarios where external services
+are secured by a Custom CA certificate, configure kapp-controller with the CA certificate data
+to prevent X.508/X.509 certificate errors.
+For detailed instructions, refer to [Deploy onto Cluster](https://{{ vars.staging_toggle }}.vmware.com/en/Cluster-Essentials-for-VMware-Tanzu/{{ vars.url_version }}/cluster-essentials/deploy.html#deploy-onto-cluster-5) in the Cluster Essentials for VMware Tanzu documentation.
+
+
+1. Create the Git secret:
+
+    Using HTTP(s) based Authentication
+    : If you are using Username and Password for authentication:
+
+
+      ```yaml
+      cat << EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: git-auth
+        namespace: tap-namespace-provisioning
+      type: Opaque
+      stringData:
+        username: GIT-USERNAME
+        password: GIT-PASSWORD
+      EOF
+      ```
+
+    Using SSH based Authentication
+    : If you are using SSH private key for authentication:
+
+
+      ```yaml
+      cat << EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: git-auth
+        namespace: tap-namespace-provisioning
+      type: Opaque
+      stringData:
+        ssh-privatekey: |
+            -----BEGIN OPENSSH PRIVATE KEY-----
+            ...
+            -----END OPENSSH PRIVATE KEY-----
+      EOF
+      ```
+
+2. Add the `secretRef` section to the `additional_sources` and the `gitops_install` section of your `tap-values.yaml` file:
+
+    Using Namespace Provisioner Controller
+    : Description
```

Comment:
Do we need the `Description` here?

Reply:
AFAIK it is required to render the tab

Reply:
I do not think we need that Description for rendering the tab. can we remove it and see once merged and we can fix it if needed after viewing it in tool?